### DISCUSSION TITLE
Feature Request: Ability to add multiple plex servers in the config list

### DISCUSSION THREAD

**Discussion by jjjonesjr33** [meisnate12/Plex-Meta-Manager#615] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
Unable to add multiple servers to a single docker instance.

**Describe the solution you'd like**
The ability to add multiple plex servers into the core config and have it connect to each one respectfully. 

**Describe alternatives you've considered**
Running multiple docker containers for each server.

**Comment by meisnate12** [comment, marked as answer]
PMM can already do this are you getting an error? check out this [example](https://github.com/meisnate12/Plex-Meta-Manager/wiki/Libraries-Attributes#library-name)

**Reply by jjjonesjr33** [reply]
That would be how you do it :D Thanks for pointing me in the right direction!

I think the only thing now would be to figure out if I can use multiple Tautulli connections.
